events/events-publisher/src/main/kotlin/uk/ac/ebi/events/service/EventsPublisherService.kt

```diff
@@ -1,30 +1,34 @@
 package uk.ac.ebi.events.service
 
 import ebi.ac.uk.extended.events.SecurityNotification
-import ebi.ac.uk.extended.events.SubmissionSubmitted
+import ebi.ac.uk.extended.events.SubmissionMessage
 import ebi.ac.uk.extended.model.ExtSubmission
 import org.springframework.amqp.rabbit.core.RabbitTemplate
 import uk.ac.ebi.events.config.BIOSTUDIES_EXCHANGE
 import uk.ac.ebi.events.config.EventsProperties
 import uk.ac.ebi.events.config.SECURITY_NOTIFICATIONS_ROUTING_KEY
+import uk.ac.ebi.events.config.SUBMISSIONS_RELEASE_ROUTING_KEY
 import uk.ac.ebi.events.config.SUBMISSIONS_ROUTING_KEY
 
 class EventsPublisherService(
     private val rabbitTemplate: RabbitTemplate,
     private val eventsProperties: EventsProperties
 ) {
-    fun securityNotification(notification: SecurityNotification) {
+    fun securityNotification(notification: SecurityNotification) =
         rabbitTemplate.convertAndSend(BIOSTUDIES_EXCHANGE, SECURITY_NOTIFICATIONS_ROUTING_KEY, notification)
-    }
 
-    fun submissionSubmitted(submission: ExtSubmission, ownerEmail: String) {
-        val instanceBaseUrl = eventsProperties.instanceBaseUrl
-        val submissionNotification = SubmissionSubmitted(
-            accNo = submission.accNo,
-            pagetabUrl = "$instanceBaseUrl/submissions/${submission.accNo}.json",
-            extTabUrl = "$instanceBaseUrl/submissions/extended/${submission.accNo}",
-            extUserUrl = "$instanceBaseUrl/security/users/extended/$ownerEmail")
+    fun submissionSubmitted(submission: ExtSubmission, ownerEmail: String) =
+        rabbitTemplate.convertAndSend(
+            BIOSTUDIES_EXCHANGE, SUBMISSIONS_ROUTING_KEY, submissionMessage(submission, ownerEmail))
+
+    fun submissionReleased(submission: ExtSubmission, ownerEmail: String) =
```

Comment:
I think the owner is not needed here. (exte already have field)

Reply:
Right, I think it comes from when it didn't have it. Fixed.